gtda/mapper/tests/test_nerve.py

```diff
@@ -22,10 +23,82 @@ def test_node_intersection(X):
     # Check if the elements of nodes defining an edge are disjoint or not:
     # If True, they are disjoint, i.e. the created edge is incorrect.
     # If all are False, all edges are correct.
-    disjoint_nodes = [set(graph['node_metadata']['node_elements'][node_1])
-                      .isdisjoint(graph['node_metadata']['node_elements']
-                                  [node_2])
+    disjoint_nodes = [set(graph.vs['node_elements'][node_1])
+                      .isdisjoint(graph.vs['node_elements'][node_2])
                       for node_1, node_2 in graph.get_edgelist()]
 
     # Check if there is a disjoint node pair given by an edge.
     assert not any(disjoint_nodes)
+
+
+@given(X=arrays(dtype=np.float, unique=True,
+                elements=floats(allow_nan=False,
+                                allow_infinity=False,
+                                min_value=-1e10,
+                                max_value=1e10),
+                shape=array_shapes(min_dims=2, max_dims=2, min_side=11)))
+def test_edge_elements(X):
+    # TODO: Replace pipe and graph by Nerve transformer
+    # TODO: Improve the Hypothesis strategy to avoid needing to hardcode the
+    # min_side to be greater than n_intervals (10 by default).
+    pipe = make_mapper_pipeline()
+    pipe_store_edge_elems = make_mapper_pipeline(store_edge_elements=True)
+
+    graph = pipe.fit_transform(X)
+    graph_ee = pipe_store_edge_elems.fit_transform(X)
```

Comment:
could we use `graph_edge_elems` here like you do above for the pipeline? i always find things like `graph_ee` are pretty cryptic when casually scrolling through code or trying to debug a failing test

Reply:
I tried to save some lines but the police found me out!